pluto/aes-proof
# fix: witnesscalc build

To verify this is all working, you can run:
```
build-circuit circuits/aes-gcm/aes-gcm-fold.circom aes-gcm-fold.bin -l node_modules
```
to generate the graph file for witness extension.

To see the constraint count for `aes-gcm-fold`, you can run:
```
circom circuits/aes-gcm/aes-gcm-fold.circom -l node_modules --r1cs
```

---

Currently addresses:
- [x] Closes  https://github.com/pluto/web-prover/issues/215

---
Need to solve:
- [ ] Makes progress towards #93
Which the following may help with:
- [x] Closes #94, addressed with 89da3ee
